crates/ethereum/evm/src/lib.rs

```diff
@@ -12,9 +12,10 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
+use alloy_primitives::{Address, U256};
 use reth_chainspec::{ChainSpec, Head};
 use reth_evm::{ConfigureEvm, ConfigureEvmEnv, NextBlockEnvAttributes};
-use reth_primitives::{transaction::FillTxEnv, Address, Header, TransactionSigned, U256};
+use reth_primitives::{transaction::FillTxEnv, Header, TransactionSigned};
 use revm_primitives::{
     AnalysisKind, BlobExcessGasAndPrice, BlockEnv, Bytes, CfgEnv, CfgEnvWithHandlerCfg, Env,
```

Comment:
@mattsse there are few instance where `Bytes` and such are being imported from `revm_primitives`. These can be imported directly from alloy instead.

Reply:
sg!